Javascript-2/Week-1/Movie App/script.js

```diff
@@ -0,0 +1,201 @@
+          star.className = starClassActive;
+        } else {
+          star.className = starClassInactive;
+        }
+      });
+    };
+  });
+
+  movieContainer.appendChild(img);
+  movieContainer.appendChild(readMoreBtn);
+  movieContainer.appendChild(starRating);
+
+  // Create element to show details about the movie
+  const movieDetails = document.createElement("div");
+  movieDetails.className = "movie-details";
+  movieDetails.style.display = "none";
+  movieDetails.style.color = "white";
+  movieDetails.innerHTML = `
+             <p><strong>Title:</strong> ${movie.title}</p>
+             <p><strong>Description:</strong> ${movie.description}</p>
+             <p><strong>Release Year:</strong> ${movie.movie_year}</p>
+             <p><strong>Director:</strong> ${movie.director}</p>
+             <p><strong>Actors:</strong> ${movie.actors.join(", ")}</p>
+         `;
+  movieContainer.appendChild(movieDetails);
+
+  // Adding the comment Section
+
+  function createCommentSection() {
+    const commentSection = document.createElement("div");
+    commentSection.innerHTML = "Comments: ";
+    commentSection.style.fontWeight = "bold";
+    commentSection.className = "comment-section";
+    const commentInput = document.createElement("input");
+    commentInput.type = "text";
+    commentInput.value = "";
+    commentInput.placeholder = "Add a Comment";
+    commentInput.style.color = "white";
+    const commentButton = document.createElement("button");
+    commentButton.className = "comment-button";
+    commentButton.innerHTML = "Submit";
+    const comments = document.createElement("div");
+    comments.className = "comments";
+    const commentUl = document.createElement("ul");
+
+    commentButton.addEventListener("click", () => {
+      const commentUl = document.createElement("ul");
+      if (commentInput.value) {
+        const comment = document.createElement("li");
+        comment.textContent = commentInput.value;
+        commentUl.appendChild(comment);
+        commentInput.value = "";
+      }
+      comments.appendChild(commentUl);
+    });
+
+    // append
+    commentSection.appendChild(commentInput);
+    commentSection.appendChild(commentButton);
+    commentSection.appendChild(comments);
+    movieDetails.appendChild(commentSection);
+  }
+  createCommentSection();
+
+  function toggle() {
+    {
+      this.classList.toggle("active");
+      /* Toggle between hiding and showing the active panel */
+      if (movieDetails.style.display === "block") {
+        movieDetails.style.display = "none";
+      } else {
+        movieDetails.style.display = "block";
+      }
+    }
+  }
+
+  readMoreBtn.addEventListener("click", toggle);
+  return movieContainer;
+}
+
+function createMoviesPage() {
+  nowStreaming.innerHTML = "";
+  movies.forEach((movie) => {
+    const movieElement = createMovieCard(movie);
+    nowStreaming.appendChild(movieElement);
+  });
+}
+createMoviesPage();
+// JS 2
```

Comment:
It's nice that you started working on the second part of the project. For the next time, please do it in a different PR. You can send your PR for the week N, checkout a new branch from it and then send a PR for the week N + 1.

```bash
git checkout main
git checkout -b javascript2-week1
# work on the assignment for the week 1
# submit the PR
git checkout -b javascript2-week2 # note that I am not starting this branch from main  but from javascript2-week1
# work on the assignment for the week 2
# submit the PR for week 2
```
Finally, when you create the PR for the week 2, choose javascript2-week1 as the base branch by click on the `base: main` button:
<img width="466" alt="image" src="https://github.com/Nusrath-Jahan/hyf-homework/assets/5467758/78bad1be-9e60-4cbb-9df8-ad57d6370b64">

Reply:
Or for simplicity, wait for the PR of week 1 to be merged before creating a new PR for week 2 (but still work on the week 2 assignment in a different branch).